src/script/api/script_list.cpp

```diff
@@ -415,7 +415,6 @@ ScriptList::ScriptList()
 
 ScriptList::~ScriptList()
 {
-	delete this->sorter;
```

Comment:
Can't the destructor be completely removed now?

Reply:
No. I tried but then it complains it doesn't know how to delete ScriptListSorter.

Reply:
Okay, that's annoying. I guess putting the whole `ScriptListSorter` into the header is a bigger problem than the empty destructor.